resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
+    name = 'Reverse complement FASTQ'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image' : 'resolwebio/common:1.1.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+
+
+    class Input:
+        """Input fields to process ReverseComplementSingle."""
+
+        reads = DataField('reads:fastq:single', label='Reads')
+
+    class Output:
+        """Output fields."""
+
+        fastq = ListField(FileField(), label='Reverse complemented FASTQ file')
+
+    def run(self, inputs, outputs):
+        """Run the analysis."""
+        name = os.path.basename(inputs.reads.fastq[0].path).strip('.fastq.gz')
```

Comment:
Very dangerous. Check what is the output of `'gaaaaaz_my_fastq.fastq.gz'.strip('fastq.gz')`.
One way of handling this::

    basename = os.path.basename(inputs.reads.fastq[0].path)
    assert basename.endswith('fastq.gz')
    name = basename[:-8]